allure-report-face/src/components/side-nav/styles.css

```diff
@@ -1,164 +1,113 @@
 @import "../../variables.css";
 
-$sidebar-full-width: 160px;
-$sidebar-collapsed: 30px;
-
 .side-nav {
+  background: $app-nav-bg;
   width: $sidebar-full-width;
   display: flex;
   flex-direction: column;
   height: 100vh;
   transition: width 0.3s;
-  & > * {
-    width: $sidebar-full-width;
-  }
+  overflow: hidden;
+
   &__head {
     margin: $gap-size 0;
     padding-bottom: $gap-size;
-    border-bottom: 1px solid $border-accent-color;
+    border-bottom: 1px solid $sidebar-separator-color;
+    transition: width 0.3s;
   }
+
   &__brand {
-    color: $text-color;
+    color: $sidebar-brand-color;
     text-decoration: none;
-  }
-  &__brand {
     display: block;
     font-size: 20px;
     line-height: 40px;
     font-weight: bold;
     background: url('allure-logo.png') no-repeat 10px center;
     padding-left: 50px;
     margin-bottom: 10px;
+    transition: all 0.3s;
+  }
+  
+  &__brand-text {
+    padding-left: $gap-size;
+    transition: opacity .2s .1s;
   }
+
   &__menu {
     margin: 0;
+    left: 0;
     padding: 0;
     list-style-type: none;
+    transition: width 0.3s;
+    text-shadow: 1px 1px 2px $sidebar-text-shadow-color;
   }
+
   &__footer {
     margin-top: auto;
     margin-bottom: $gap-size;
+    padding-left: 0;
+    padding-right: 0;
   }
+
   &__item {
     display: block;
-    font-size: 16px;
+    font-size: 14px;
   }
+
   &__link, &__collapse {
     display: block;
+    line-height: 24px;
     color: $text-muted-color;
     text-decoration: none;
     transition: color 0.1s ease-in;
     padding: 0.8em 0;
     &_active {
-      color: $text-color;
+      color: $sidebar-active-text-color;
+      border-right: 4px solid $sidebar-active-accent-color;
     }
     &:hover {
-      color: $text-color;
+      color: $sidebar-hover-text-color;
     }
   }
   &__icon {
+    position: absolute;
+    left: $gap-size;
     font-size: 22px;
-    line-height: 24px;
     vertical-align: middle;
     transition: top 0.3s ease-in;
     margin: 0 5px;
   }
-  &__report {
-    display: none;
-  }
-  &__info {
-    line-height: 1.3;
-    position: relative;
-    &:after {
-      position: absolute;
-      content: ' ';
-      left: 0;
-      right: 0;
-      background: linear-gradient(to left, $body-bg, color($body-bg a(0)));
-      transition: right 0.3s;
-    }
-    dt {
-      color: $text-muted-color;
-    }
-    dd {
-      margin-left: 0;
-      margin-bottom: 0.5em;
-    }
-  }
-  &__flag {
-    display: inline-block;
-    width: 16px;
-    height: 12px;
-    background-size: cover;
-    background-position: center;
-    position: relative;
-    top: -2px;
-    vertical-align: middle;
-    &_lang_en {
-      background-image: url(~flag-icon-css/flags/4x3/gb.svg);
-    }
-    &_lang_ru {
-      background-image: url(~flag-icon-css/flags/4x3/ru.svg);
-    }
-    &_lang_ptbr {
-      background-image: url(~flag-icon-css/flags/4x3/br.svg);
```

Comment:
If you are going to remove flag icons at all, please do not forget to remove `flag-icon-css` from dependencies